.github/workflows/release-nightly.yaml

```diff
@@ -71,7 +71,7 @@ jobs:
       - name: Update versions
         shell: bash
         run: |
-          version=0.0.0-nightly-$(date +"%y%m%d")
+          version=0.0.0-nightly

```

Comment:
Please revert to including the date in the version, so people can tell when they got their release. 🙏

Reply:
Yeah that's what @HarryET mentioned as well, I was a little overzealous at first when I went about removing the date from everything 😆

Reply:
Does this also apply to the change on line 164?